### ISSUE TITLE
Cannot zoom in PointCloud viewer on mobile

### ISSUE THREAD

**Issue by jmachuca77** [dheera/rosboard#107]
Is there an easy fix for allowing zoom to work on a mobile device? I can spin the view but I cant zoom or move the view.

**Comment by dheera** [comment]
Good point. Might need to implement a tap-and-drag gesture like Google maps has.
